### PR TITLE
update 'Get in touch' links to Rocket and forums

### PR DESCRIPTION
**Pull request by Ads20000**
Previously the links in the 'Get in touch' section pointed to IRC and the mailing list but now Rocket and the forums are used, this commit updates the links to the new mediums of communication.

See my merged PR for the snapcraft repository [here](https://github.com/snapcore/snapcraft/pull/1268). I should've merged it into the snapd repository earlier so sorry about that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ If you have found an issue with the application, please [file a bug](https://bug
 
 ## Get in touch
 
-We're friendly! Talk to us on [IRC](https://webchat.freenode.net/?channels=snappy)
```
Comment (zyga) [inline README.md]: I think we want both links. Forum, IRC and Rocket chat are all used equally.

Comment (Ads20000) [inline README.md]: Oh, IRC is still used for `snapd` development? Well you're the one who would know! :P

Comment (Ads20000) [inline README.md]: What order should they be listed?

Comment (zyga) [inline README.md]: I'm fine with any order really :-)

Comment (Ads20000) [inline README.md]: I was going to put Rocket Chat first, then checked the Rocket Chat activity after noticing I'd got the channel link wrong and decided to put IRC first :P I'm guessing you'd still like to use Rocket Chat as an active development channel so I've left it in?

**Pull request merged**